azurelinuxagent/common/conf.py

```diff
@@ -508,7 +508,7 @@ def get_ga_updates_enabled(conf=__conf__):
     If True, the agent go through update logic to look for new agents otherwise it will stop agent updates.
     NOTE: This option is needed in e2e tests to control agent updates.
     """
-    return conf.get_switch("GAUpdates.Enabled", True)
+    return conf.get_switch("Debug.EnableGAUpdates", True)
```

Comment:
The description in the comment needs improving. From reading it, it is hard to tell how it is different to AutoUpdate.Enabled

Reply:
The description for Debug.EnableGAVersioning also needs improving